sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_set/feature_set_materialization_metadata.py

```diff
@@ -69,17 +69,30 @@ def __init__(
         self.tags = tags
 
     @classmethod
-    def _from_rest_object(cls, obj: RestFeaturesetJob) -> "FeatureSetMaterializationMetadata":
+    def _from_rest_object(cls, obj: RestJobBase) -> "FeatureSetMaterializationMetadata":
```

Comment:
Just make sure that We don't use _from_rest_object for RestFeaturesetJob anywhere else in the code.

Reply:
Did a repo wide search, the RestFeaturesetJob  only apears in this file, and it's updated to RestJobBase.